README.md

```diff
@@ -21,68 +20,70 @@ A minimalistic Flutter game engine.
-Feel free to choose the one that better suits your needs.
+## Documentation
 
----
+The full documentation for Flame can be found on the 
+[docs.flame-engine.org](https://docs.flame-engine.org/).
 
-## Documentation
+To change the version of the documentation, use the version selector noted with `version:` in the
+top of the page.
 
 Note that the documentation in the main branch of this repo is newer than the latest released
 version.
 
-Here you can find the documentation for different versions:
-- Latest stable version: [Flame-engine website](https://flame-engine.org/)
-- Latest stable version: [GitHub docs](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
-- Latest v1.0.0 version: [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-releasecandidate.14/doc)
+Other useful links:
+ - [The official Flame site](https://flame-engine.org/).
+ - [Examples](https://examples.flame-engine.org/) of most features which can be tried out from your browser.
+ - [Tutorials](https://tutorials.flame-engine.org/) - A simple tutorial to get started.
+ - [API Reference](https://pub.dev/documentation/flame/latest/) - The generated dartdoc API reference.
 
-The complete documentation can be found [here](https://github.com/flame-engine/flame/tree/main/doc).
-
-We provide many examples of different features which can be tried out from your browser
-[here](https://flame-engine.github.io/flame/). You can also check the examples source code [here](https://github.com/flame-engine/flame/tree/main/examples).
-
-The official site for Flame, which also contains the documentation can be found
-[here](https://flame-engine.org/).
 
 ## Help
 
-We have a Flame help channel on Blue Fire's Discord, join it [here](https://discord.gg/5unKpdQD78).
-
-We also have a [FAQ](FAQ.md), so please search for your question there first.
-
-## Goals
-
-The goal of this project is to provide a complete set of out-of-the-way solutions for the common
-problems every game developed in Flutter will share.
+We have a Flame community and help channel on [Blue Fire's Discord](https://discord.gg/5unKpdQD78).
+
+If you are more comfortable with StackOverflow you can create a question with the
+[Flame tag](https://stackoverflow.com/questions/tagged/flame) and we'll reply as soon as we can.
+
+
+## Features
+
+The goal of Flame is to provide a complete set of out-of-the-way solutions for the common problems
+every game developed in Flutter will share.
+
+Currently Flame provides:
+ - A game loop.
+ - A component/object system (FCS).
+ - Effects and particles.
+ - Collision detection.
+ - Gesture and input support.
+ - Images, animations, sprites and sprite sheets.
+ - Other utilities to make development easier.
+
+On top of those features we offer something that we call bridge libraries. Through these libraries
+we offer Flame components and bindings to other packages. Currently we have bridge
```

Comment:
```suggestion
we can bind to other packages and provide more game-related functionalities that normally are out of the scope of the main engine. Currently we have bridge
```

Maybe give the reason why this exists?

Reply:
The reason is more complicated than that though. I thought it'd be better just to not go into depth about why on the readme.

Reply:
I disagree, the reason is simple. Integrate with other functionalities. That there is a deeper technical reason is not really relevant for the end-user right?

Reply:
Check Luan's update, I think that one was pretty good :)